web/src/app/components/GameOverWindow.tsx

```diff
@@ -0,0 +1,25 @@
+import { Button } from '@mantine/core';
+import React from 'react';
+import PlayerState, { Player } from '../interfaces/PlayerState';
+
+interface GameOverProps {
+    isHost: boolean;
+    playerID: string;
+    players: PlayerState;
+    onPlayAgain: () => void;
+};
+
+export default function GameOverWindow({isHost, playerID, players, onPlayAgain} : GameOverProps)  {
+
+  const isWinner: boolean = players[playerID].place === 0;
+  const winner: Player | undefined = Object.values(players).find(p => p.place === 0);
+
+  return (
+    <>
+        <div className="flex flex-col items-center space-y-4">
+            <p className="text-2xl">{isWinner ? "you win!" : `${winner?.username} wins!`}</p>
```

Comment:
Does "You win!" look better?

Reply:
i tried it and liked the lowercase better

Reply:
Lowercase looks bad no? I just looked at it